en_ewt-ud-train.conllu

```diff
@@ -208850,7 +208850,7 @@
 # text = HEAVEN ON EARTHHHHHHH!!!!
 1	HEAVEN	heaven	NOUN	NN	Number=Sing	0	root	0:root	_
 2	ON	on	ADP	IN	_	3	case	3:case	_
-3	EARTHHHHHHH	Earth	PROPN	NNP	Number=Sing	1	nmod	1:nmod:on	SpaceAfter=No
+3	EARTHHHHHHH	Earth	PROPN	NNP	Number=Sing|Typo=Yes	1	nmod	1:nmod:on	SpaceAfter=No|CorrectForm=earth
```

Comment:
I would guess this is stylistic spelling (expressive lengthening), not exactly a typo

Reply:
(Should CorrectForm be in all-caps?)

Reply:
Probably yes

Reply:
should still have `CorrectForm`